### DISCUSSION TITLE
Links

### DISCUSSION THREAD

**Discussion by six6out** [community/community#21852] [Other Feature Feedback, Questions, & Ideas]

**Comment by Wabri** [comment]
<p>Hi <a class="mention" href="/javasight">@javasight</a>,</p>
<p>Welcome to the Github Community Forum!</p>
<p>Same question here I think: <a href="https://github.community/t/github-pages-published-but-still-404/10843/2">GitHub-pages-published-but-still-404</a> </p>
<p>This is your login page? <a href="https://lunatic-coding.github.io" rel="nofollow noopener">lunatic-coding.github.io</a> </p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/d/d9f5ecd0c911a8e16bc42ef68f6f10d0a83118ff.png" data-download-href="/uploads/short-url/v6avbUS7W71HelAQmCGQusskm6j.png?dl=1" title="2019-12-26-16-20-11_422x540.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/12409541/181067130-0dc74077-02c2-4bdb-9ccb-2c01d927d2b1.png" alt="2019-12-26-16-20-11_422x540.png" data-base62-sha1="v6avbUS7W71HelAQmCGQusskm6j" width="390" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/d/d9f5ecd0c911a8e16bc42ef68f6f10d0a83118ff_2_390x500.png, https://aws1.discourse-cdn.com/github/original/2X/d/d9f5ecd0c911a8e16bc42ef68f6f10d0a83118ff.png 1.5x, https://aws1.discourse-cdn.com/github/original/2X/d/d9f5ecd0c911a8e16bc42ef68f6f10d0a83118ff.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/d/d9f5ecd0c911a8e16bc42ef68f6f10d0a83118ff_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">2019-12-26-16-20-11_422x540.png</span><span class="informations">422×540 83.3 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>-Gabriele-</p>
<p>Mark helpfull posts with<span> </span>Accept as Solution<span> </span>to help other users locate important info. Don’t forget to give<span> </span>Kudos<span> </span>for great contents!</p>

**Comment by six6out** [comment, marked as answer]
<p>Yea. I already solved it, just forgot to post.</p>
<p>&lt;a href=“example.html” onclick=“window.open(‘example.html’, ‘_self’);”&gt;</p>
<p>  Example</p>
<p>&lt;/a&gt;</p>
